pages/common/comby.md

```diff
@@ -0,0 +1,20 @@
+# comby
+
+> Tool for structural code search and replace that supports many languages.
+> More information: <https://github.com/comby-tools/comby>.
+
+- Match and rewrite templates, and print changes:
+
+`comby '{{assert_eq!(:[a], :[b])}}' '{{assert_eq!(:[b], :[a])}}' {{.rs}}`
+
+- Match and rewrite with rewrite properties:
+
+`comby '{{assert_eq!(:[a], :[b])}}' '{{assert_eq!(:[b].Capitalize, :[a])}}' {{.rs}}`
+
+- Match and rewrite in-place:
+
+`comby -in-place {{match_pattern}} {{rewrite_pattern}}`
```

Comment:
```suggestion
`comby -in-place '{{match_pattern}}' '{{rewrite_pattern}}'`
```

Reply:
since simple examples is already provided above for commands without options, will it be repetitive to give more examples here?

Reply:
Yes it will be repetitive. But on the other hand you can choose one appropriate example which is used when you want to demonstrate some options of command and how they affects command behavior. In case of `sed` previous sentence means - select some sed's command (**s**ubstitute for instance) and use it with some option (`-E` for instance) and without it and show the difference in BRE and ERE syntax on simple replacement. 😁